pkg/virt-controller/watch/vm.go

```diff
@@ -73,6 +73,7 @@ const (
 	failedCreateCRforVmErrMsg             = "Failed to create controller revision for VirtualMachine."
 	failedProcessDeleteNotificationErrMsg = "Failed to process delete notification"
 	failureDeletingVmiErrFormat           = "Failure attempting to delete VMI: %v"
+	memoryDumpFailed                      = "Memory dump failed"
```

Comment:
In order to be consistent with the variable name, I'd call it `failedMemoryDump`